WiseBuy/View Controllers/DetailsViewController.h

```diff
@@ -6,11 +6,14 @@
 //
 
 #import <UIKit/UIKit.h>
+#import "AppDeal.h"
```

Comment:
You don't need to explicitly import this to use it in the header file as a property. You can use `@class AppDeal` to forward declare the class for the header file (and actually import it in the .m file)